doc/src/learn/cryptography/sui-move-signatures.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Sui On-Chain Signatures Verification in Move
+---
+
+Move contracts in Sui supports verifications for several signature schemes on chain. Note that not all signatures supported in on-chain verification are supported as user signature verification. See [Sui Signatures](sui-signatures.md#user-signature) for valid signatures to be used for transaction execution. 
```

Comment:
1. support (no s)
2. for valid signatures -> for valid signature schemes for transaction authorization.